Re-enable working integration tests

Neverlord opened the pull request:
Two tests are still disabled:
- `Node Basic mrt`: seems to produce unstable results; I get baseline mismatches even after updating via `-u`
- `Manual 2`: flat out fails for me

All other tests were enabled and I've added baselines where needed.

mavam commented:
> All other tests were enabled

Did you also make sure that the results are *correct*? I don't recall why they were turned off, so we should be certain that we validated the output.

Neverlord commented:
Fair enough. I've double-checked selected queries for the four integration tests and everything LGTM.

Here's the full report for due diligence (I've removed `vast.db` whenever importing new data):

# Server Basic bgpdump

The test query has no result:

```
$ gunzip -c data/bgpdump/updates.20180530.bgpdump.gz | grep -E '\|114\.' | wc
       0       0       0
```

So I've added a second test case that has results:

```
$ gunzip -c data/bgpdump/updates.20180530.bgpdump.gz | grep '103.111.8'
BGP4MP|1527638481|A|212.25.27.44|8758|103.111.8.0/24|8758 9498 58587 58689 137505|IGP|212.25.27.44|0|0|9498:1 9498:91 9498:9344 9498:58587 34111:9498 34307:50172 34911:9498 40505:9498 8758:215 8758:301|NAG||
BGP4MP|1527638541|A|212.25.27.44|8758|103.111.8.0/23|8758 9498 58587 58689 137505|IGP|212.25.27.44|0|0|9498:1 9498:91 9498:9344 9498:58587 34111:9498 34307:50172 34911:9498 40505:9498 8758:215 8758:301|NAG||
BGP4MP|1527638665|A|213.200.87.254|3257|103.111.8.0/24|3257 6453 58587 58689 137505|IGP|213.200.87.254|0|10|3257:8070 3257:30114 3257:50001 3257:53900 3257:53902|NAG||
BGP4MP|1527638677|A|213.200.87.254|3257|103.111.8.0/24|3257 6453 58587 58689 137505|IGP|213.200.87.254|0|10|3257:8024 3257:30107 3257:50001 3257:53100 3257:53101|NAG||
```

VAST gives the proper results:

```
$ vast export -N ascii '103.111.8.42 in prefix'
<2018-05-30+00:01:21.0, 212.25.27.44, 8758, 103.111.8.0/24, [8758, 9498, 58587, 58689, 137505], nil, "IGP", 212.25.27.44, 0, 0, "9498:1 9498:91 9498:9344 9498:58587 34111:9498 34307:50172 34911:9498 40505:9498 8758:215 8758:301", "NAG", nil>
<2018-05-30+00:02:21.0, 212.25.27.44, 8758, 103.111.8.0/23, [8758, 9498, 58587, 58689, 137505], nil, "IGP", 212.25.27.44, 0, 0, "9498:1 9498:91 9498:9344 9498:58587 34111:9498 34307:50172 34911:9498 40505:9498 8758:215 8758:301", "NAG", nil>
<2018-05-30+00:04:25.0, 213.200.87.254, 3257, 103.111.8.0/24, [3257, 6453, 58587, 58689, 137505], nil, "IGP", 213.200.87.254, 0, 10, "3257:8070 3257:30114 3257:50001 3257:53900 3257:53902", "NAG", nil>
<2018-05-30+00:04:37.0, 213.200.87.254, 3257, 103.111.8.0/24, [3257, 6453, 58587, 58689, 137505], nil, "IGP", 213.200.87.254, 0, 10, "3257:8024 3257:30107 3257:50001 3257:53100 3257:53101", "NAG", nil>
```

# Query Operators

This test has a couple of queries, so I've picket two that I can easily verify.

## `service == smtp`

Comparing `grep` to `vast`:

```
$ gunzip -c data/zeek/conn.log.gz | grep smtp > out1.txt
$ vast export -N zeek 'service  == "smtp"' | bro-cut > out2.txt
$ wc -l out1.txt 
      21 out1.txt
$ wc -l out2.txt 
      21 out2.txt
$ if diff out1.txt out2.txt ; then echo "ok" ; else echo "files differ" ; fi
5c5
< 1258565880.189257	U6EoRx2yUyl	192.168.1.105	49219	192.168.1.1	25	tcp	smtp	0.022985	4429	799	SF	-	0	ShAadDfF	18	5161	30	2011	(empty)
---
> 1258565880.189257	U6EoRx2yUyl	192.168.1.105	49219	192.168.1.1	25	tcp	smtp	0.0229850	4429	799	SF	-	0	ShAadDfF	18	5161	30	2011	(empty)
7c7
< 1258566123.706408	zkdnQLD4rx7	192.168.1.103	1806	192.168.1.1	25	tcp	smtp	0.033244	4371	1370	SF	-	0	ShAdDafF	17	5059	28	2498	(empty)
---
> 1258566123.706408	zkdnQLD4rx7	192.168.1.103	1806	192.168.1.1	25	tcp	smtp	0.0332440	4371	1370	SF	-	0	ShAdDafF	17	5059	28	2498	(empty)
10c10
< 1258567289.262109	GxYIFhoLgza	192.168.1.102	1405	192.168.1.1	25	tcp	smtp	0.021483	5796	632	SF	-	0	ShAdDafF	19	6564	28	1760	(empty)
---
> 1258567289.262109	GxYIFhoLgza	192.168.1.102	1405	192.168.1.1	25	tcp	smtp	0.0214830	5796	632	SF	-	0	ShAdDafF	19	6564	28	1760	(empty)
13c13
< 1258568059.128662	ufFpSyrwjwd	192.168.1.103	1836	192.168.1.1	25	tcp	smtp	0.031994	3372	1370	SF	-	0	ShAdDafF	16	4020	26	2418	(empty)
---
> 1258568059.128662	ufFpSyrwjwd	192.168.1.103	1836	192.168.1.1	25	tcp	smtp	0.0319940	3372	1370	SF	-	0	ShAdDafF	16	4020	26	2418	(empty)
15,16c15,16
< 1258568738.108255	Rdz0JuTWH7h	192.168.1.105	49561	192.168.1.1	25	tcp	smtp	0.032979	4334	1466	SF	-	0	ShAadDfF	17	5026	28	2598	(empty)
< 1258574141.027462	K1ETvrXVgr	192.168.1.104	1572	192.168.1.1	25	tcp	smtp	0.438735	1021	308	SF	-	0	ShAdDafF	10	1429	17	996	(empty)
---
> 1258568738.108255	Rdz0JuTWH7h	192.168.1.105	49561	192.168.1.1	25	tcp	smtp	0.0329790	4334	1466	SF	-	0	ShAadDfF	17	5026	28	2598	(empty)
> 1258574141.0274620	K1ETvrXVgr	192.168.1.104	1572	192.168.1.1	25	tcp	smtp	0.438735	1021	308	SF	-	0	ShAdDafF	10	1429	17	996	(empty)
21c21
< 1258583614.298059	IcCYHXxqpC5	192.168.1.102	1911	192.168.1.1	25	tcp	smtp	1.025109	7079	1370	SF	-	0	ShAdDafF	20	7887	29	2538	(empty)
---
> 1258583614.298059	IcCYHXxqpC5	192.168.1.102	1911	192.168.1.1	25	tcp	smtp	1.0251090	7079	1370	SF	-	0	ShAdDafF	20	7887	29	2538	(empty)
files differ
```

If you look closely, the only difference between the raw zeek log and the VAST output is that VAST prints a couple of extra `0`s for some floating point numbers. Not character-perfect, but the results match.

## `missed_bytes != 0`

Output via `awk`:

```
$ gunzip -c data/zeek/conn.log.gz | bro-cut | awk -F $'\t' '{ if ($14 != 0) print $14 }' > out1.txt
$ vast export -N zeek 'missed_bytes != 0' | bro-cut | awk -F $'\t' '{ if ($14 != 0) print $14 }' > out2.txt
$ wc -l out1.txt 
      58 out1.txt
$ wc -l out2.txt 
      58 out2.txt
$ if diff out1.txt out2.txt ; then echo "ok" ; else echo "files differ" ; fi
ok
```

## Manual 1 / Manual 1 node

The two tests are identical. The only difference is that the latter uses `-N` on import/export.

```
$ gunzip -c data/pcap/example.pcap.gz > example.pcap
$ tcpdump -ttttnnr example.pcap > example.pcap.txt
$ grep -o -E '192\.168\.168.[0-9]+' example.pcap.txt > out1.txt
$ vast export -N ascii ':addr in 192.168.168.0/24' | grep -o -E '192\.168\.168.[0-9]+' > out2.txt
$ wc -l out1.txt 
    1008 out1.txt
$ wc -l out2.txt 
    1008 out2.txt
$ if diff out1.txt out2.txt ; then echo "ok" ; else echo "files differ" ; fi
ok
```

mavam commented:
Very convincing due diligence, thanks for taking the extra time and presenting it here in great detail.
